pulsar-io/kinesis/src/main/java/org/apache/pulsar/io/kinesis/BaseKinesisConfig.java

```diff
@@ -38,7 +37,7 @@
     private String awsEndpoint;
     
     @FieldDoc(
-        required = true,
+        required = false,
```

Comment:
it should be either/or for this and endpoint. if region is set, you should be able to leave endpoint blank